src/main/ks-installer/README.md

```diff
@@ -0,0 +1,105 @@
+# Ks-installer Helm Chart
+
+## Introduction
+
+The helm chart of KubeSphere, supports installing KubeSphere on existing Kubernetes.
```

Comment:
Add the brief introduction of KubeSphere here.

Reply:
You can also introduce the relationship between ks-installer and helm chart of KubeSphere.

Reply:
The modification has been completed. @FeynmanZhou